src-gui/ui/src/app/common/components/islmaintenancemodal/islmaintenancemodal.component.ts

```diff
@@ -10,22 +11,41 @@ export class IslmaintenancemodalComponent implements OnInit {
 
   title: any;
   content: any;
+  descriptionValue: any;
   isEvacuate:boolean=false;
-  isMaintenance:boolean;
-
+  isMaintenance: boolean;
+  isDescription: boolean;
+  DescriptionForm: FormGroup;
```

Comment:
It could be better to start the variable name with the lower case for the first letter.